docs/cn/instructions/quickstart-cn.md

```diff
@@ -227,9 +229,9 @@ ROCKETMQ_HOME=D:\rocketmq
 >  abortFile=你的store目录/abort
 >  ```
 
-![](../../images/quickstart/rocketmq-namesrv-ev.jpg)
+![](../../images/quickstart/rocketmq-namesrv-ev.png)
 
-![](../../images/quickstart/rocketmq-broker-ev.png)
+![](../../images/quickstart/rocketmq-broker-ev.jpg)
```

Comment:
Can we use `.png` here

Reply:
I will unify all the pictures into. PNG format.